### PR TITLE
EDU-177 publish only on tags

### PR DESCRIPTION
**Pull request by ahelmberger**
closes https://educandu.atlassian.net/browse/EDU-177

### CODE REVIEW

**Comment by laurentiu-ilici** [comment]
So now the title of the PR doesn't matter anymore: "Feature/edu 177 publish only on tags". You usually do EDU-xxx.

**Comment by ahelmberger** [comment]
> So now the title of the PR doesn't matter anymore: "Feature/edu 177 publish only on tags". You usually do EDU-xxx.

Hm, Github created it from the branch name. I changed it manually to "EDU-177". It's strange bacause usually I didn't have to do this.

File: README.md [hunk 1]
```diff
@@ -1,3 +1,17 @@
+* Merge pull request #80 from educandu/feature/[EDU-150](https://educandu.atlassian.net/browse/EDU-150)-fix-image-tiles ([16d4e72](https://github.com/educandu/educandu/tree/16d4e72))
+* [EDU-150](https://educandu.atlassian.net/browse/EDU-150) Optimize the migration ([dee8a02](https://github.com/educandu/educandu/tree/dee8a02))
+* [EDU-150](https://educandu.atlassian.net/browse/EDU-150) Fix translations ([b8589ea](https://github.com/educandu/educandu/tree/b8589ea))
+* [EDU-150](https://educandu.atlassian.net/browse/EDU-150) Add migration for image tile content ([73977a4](https://github.com/educandu/educandu/tree/73977a4))
+* [EDU-150](https://educandu.atlassian.net/browse/EDU-150) Add not found on response to redirecting with a document key ([2a62708](https://github.com/educandu/educandu/tree/2a62708))
+* [EDU-150](https://educandu.atlassian.net/browse/EDU-150) Change link type from article to internal ([ed11685](https://github.com/educandu/educandu/tree/ed11685))
+
```
Comment (laurentiu-ilici) [inline README.md:7]: So many empty lines :)

**Review by laurentiu-ilici — APPROVED** [review]
Looks good :D

File: README.md (hunk 1 above)
Comment (ahelmberger) [reply README.md:7]: ....and so much garbage above, will remove it, sorry, that was not supposed to be there

File: .github/workflows/publish.yml [hunk 2]
```diff
@@ -0,0 +1,44 @@
...
+        run: ./node_modules/.bin/gulp verifySemverTag --tag ${{ steps.tag.outputs.tag }}
+      - name: Set package version
+        run: npm version --allow-same-version --git-tag-version=false ${{ steps.tag.outputs.tag }}
+      - name: Build library
+        run: ./node_modules/.bin/gulp build
+      - name: Authenticate with npm
+        run: npm set //registry.npmjs.org/:_authToken $NPM_AUTH_TOKEN
+      - name: Publish to npm
+        run: npm publish
+      - name: Create Github release and tag JIRA issues
```
Comment (adrianaluchian) [inline .github/workflows/publish.yml:43]: Nice!

**Pull request merged**